capstone/src/main/java/com/step902020/capstone/IndividualController.java

```diff
@@ -304,4 +309,58 @@ public String uploadImage(CurrentUser user) throws IOException{
   public @ResponseBody byte[] getImage(CurrentUser user) throws IOException {
     return gcsstore.serveImage("step90-2020", "step90-2020.appspot.com", user.getEmail());
   }
+
+  /**
+   * sets the recommender to be used for creating recommendations
+   * @param newRecommender recommender to be used
+   */
+  public void setRecommender(Recommender newRecommender) {
+    recommender = newRecommender;
+  }
+
+  /**
+   * Add new review to event
+   * @param user current user
+   * @param text review's text
+   * @param orgId organization's datastore id
+   * @return updated review list
+   */
+  @PostMapping("/add-org-review")
```

Comment:
Should this be in individual controller if it modifies the organization entity's data?

Reply:
Also, if we move functions into the controller of who triggers the function, would that mean we should move everything into the org or individual controllers? So add/remove/like/edit event reviews should also be in the individual controller and add/edit events should be in the organization controller?

Reply:
Hmm, yeah that is true. The reason why I couldn't move it into the organizationtests was because to do these tests you need the current user to be an individual, but for the rest of the tests I need the current user to be an organization.
We can also move it back to individual, or maybe moving it into review tests?